### PR TITLE
Use init-only properties

### PR DESCRIPTION
**Pull request by oschwald**

### CODE REVIEW

File: MaxMind.MinFraud/Request/CreditCard.cs [hunk 1]
```diff
@@ -49,23 +53,6 @@ public sealed class CreditCard
             string? token = null
         )
         {
-            if (issuerIdNumber != null && !IssuerIdNumberRe.IsMatch(issuerIdNumber))
```
Comment (horgh) [inline MaxMind.MinFraud/Request/CreditCard.cs:52]: I'm guessing this validation will still run if we create the object in the old way too?

**Review by horgh — CHANGES_REQUESTED** [review]
Pretty nice syntax. Is this worth mentioning in the changelog?

**Comment by oschwald** [comment]
Huh, I must have lost the release notes in a rebase somehow. Hopefully I didn't lose anything else.

File: MaxMind.MinFraud/Request/CreditCard.cs (hunk 1 above)
Comment (oschwald) [reply MaxMind.MinFraud/Request/CreditCard.cs:52]: Yep, all of the constructors initialize through the properties.

**Comment by oschwald** [comment]
It turns out there were a few commits that I failed to push from Windows. I've added them

**Review by horgh — APPROVED** [review]

**Pull request merged**